integreat_cms/cms/constants/time_range_events.py

```diff
@@ -0,0 +1,24 @@
+"""
+This module contains all string representations of time_range_events filter options, used by :class:`~integreat_cms.cms.forms.events.event_filter_form.EventFilterForm` and
+:class:`~integreat_cms.cms.views.events.event_list_view.EventListView`.
+
+The module also contains a constant :attr:`~integreat_cms.cms.constants.all_day.DATATYPE`, which contains the type of the constant
+values linked to the strings and is used for correctly instantiating :class:`django.forms.TypedMultipleChoiceField`
+instances in :class:`~integreat_cms.cms.forms.events.event_filter_form.EventFilterForm`.
+"""
+from django.utils.translation import ugettext_lazy as _
+
+
+#: This docstring is overridden by "alias of builtins.int"
+DATATYPE = int
+
+#: Only events which are all day long
+UPCOMING_EVENTS = 1
+#: Exclude events which are all day long
+PAST_EVENTS = 2
```

Comment:
```suggestion
UPCOMING_EVENTS = "UPCOMING_EVENTS"
#: Exclude events which are all day long
PAST_EVENTS = "PAST_EVENTS"
```
I know you just copied + pasted the existing filters, but I think the existing solution is a little bit flawed. In my opinion, it doesn't make sense to cast the values to ints manually, if we can just use more readable strings. This makes debugging a little bit easier.